.github/scripts/releases.py

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from github import Github
+
+# How to use this script:
+# Set the following env vars:
+#   * GH_RELEASE_TOKEN
+#   * LAMBDA_RUNTIME (optional)
+#   * INITCONTAINER_RUNTIME (optional)
+#   * AGENT_VERSION
+#   * DRY_RUN (optional) set to 1 to run the flow without creating the release tag.
+#
+# Lambda runtime options:
+# nodejs
+# python
+# ruby
+# dotnet
+#
+# Initcontainer runtime options:
+# nodejs
+# python
+# ruby
+# java
+# dotnet
+
+DRY_RUN = os.environ.get("DRY_RUN", "").lower() in ("on", "true", "1")
+
+
+def run_container_releases():
+    gh = Github(os.environ["GH_RELEASE_TOKEN"])
+    lambda_runtime = os.environ.get("LAMBDA_RUNTIME")
+    initcontainer_runtime = os.environ.get("INITCONTAINER_RUNTIME")
+    latest_version = os.environ["AGENT_VERSION"]
+
+    lambda_repo = gh.get_repo("newrelic/newrelic-lambda-layers")
+    initcontainer_repo = gh.get_repo("newrelic/newrelic-agent-init-container")
+
+    if lambda_runtime:
+        create_release(lambda_repo, lambda_runtime, latest_version)
+    if initcontainer_runtime:
+        create_release(initcontainer_repo, initcontainer_runtime, latest_version)
+
+
+def create_release(repo, runtime, latest_version):
+    tag_name = f"{latest_version}.0_{runtime}"
+    release_name = f"New Relic {runtime} Agent {latest_version}.0"
```

Comment:
This originally had logic to have release names and release tags as different strings. For Python the only difference was capitalization, but for Node and .NET it was `Node -> nodejs`, and `.NET -> dotnet`.

Why not make constants for `TAG_SUFFIX="python"` and `RUNTIME_NAME="Python"`. Then maybe use bool constants to toggle on/off the lambda and initcontainer releases?

Reply:
oooh you're totally right! I was trying to simplify those out but they were used for the release name-I'll put those back I guess. Instead of a boolean I was thinking each team could just set the value of those and that way the work flow could be shared across all teams-I need to look into that more though to see if we can do that.